eadf/importers.py

```diff
@@ -186,3 +189,53 @@ def fromAngleListData(
     )
 
     return EADF(arrInter, arrAzi, arrCoEle, arrFreqData, arrPos)
+
+def loadData(
+    path: str,
+    numPor = 0,
```

Comment:
why is it the number of ports, and not an array of port indices? why is this feature only there for the antenna ports and not also for frequency and polarisation? also the specification of the default value is not pep8-conform.

Reply:
Agree that the user should be able to choose the specific antenna ports, polarization and frequency points.